bublik/interfaces/api_v2/config.py

```diff
@@ -188,8 +188,8 @@ def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
 
     @auth_required(as_admin=True)
-    @action(detail=False, methods=['get'])
-    def schema(self, request, *args, **kwargs):
+    @action(detail=False, methods=['get'], url_path='schema')
+    def get_schema(self, request, *args, **kwargs):
```

Comment:
It is unclear why happens before the patch. Is everything broken? If so, it should be squashed into previous changeset where drf_spectacular is added.

Reply:
The conflict did not break the application but prevented correct schema generation.
No clear error message was raised, making the issue difficult to debug.
I would prefer this change to be explicit for clarity and traceability

Clarified commit description